index.html

```diff
@@ -41,7 +41,180 @@
+              <li>html</li>
+            </ul> 
+        </div>
+        <div id="see-project-btn">
+            <p id="see-project">See Project</p>
+        </div>
+        </div>
+    </div>
+    <div class="card">
+        <div class="img-Plaholder"></div>
+        <div class="project-detail">
+        <p id="card-text">Multi-Post Stories Gain+Glory</p>
+        <div class="work-category">
+            <ul id="work-list">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavaScript</li>
+              <li>html</li>
+            </ul> 
+        </div>
+        <div id="see-project-btn">
+            <p id="see-project">See Project</p>
+        </div>
+        </div>
+    </div>
+    <div class="card">
+        <div class="img-Plaholder"></div>
+        <div class="project-detail">
+        <p id="card-text">Multi-Post Stories Gain+Glory</p>
+        <div class="work-category">
+            <ul id="work-list">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavaScript</li>
+              <li>html</li>
+            </ul> 
+        </div>
+        <div id="see-project-btn">
+            <p id="see-project">See Project</p>
+        </div>
+        </div>
+    </div>
+    <div class="card">
+        <div class="img-Plaholder"></div>
+        <div class="project-detail">
+        <p id="card-text">Multi-Post Stories Gain+Glory</p>
+        <div class="work-category">
+            <ul id="work-list">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavaScript</li>
+              <li>html</li>
+            </ul> 
+        </div>
+        <div id="see-project-btn">
+            <p id="see-project">See Project</p>
+        </div>
+        </div>
+    </div>
+    <div class="card">
+        <div class="img-Plaholder"></div>
+        <div class="project-detail">
+        <p id="card-text">Multi-Post Stories Gain+Glory</p>
+        <div class="work-category">
+            <ul id="work-list">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavaScript</li>
+              <li>html</li>
+            </ul> 
+        </div>
+        <div id="see-project-btn">
+            <p id="see-project">See Project</p>
+        </div>
+        </div>
+    </div>
+    </section>
+    <section class="about">
+        <div class="about-me">
+        <div class="about-title">
+            <p id="about-text">About me</p>
+            <div class="about1">
+            <img src="images/About1.png" alt="About-Frame" />
+        </div>
+        </div>
+        <div class="about-detail">
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+        </div>
+        <div id="get-my-resume-btn">
+            <p id="get-my-resume">Get My Resume</p>
```

Comment:
Kindly use a `<button>`  for this. This is for semantic purposes and so that screen readers can understand your page